include/sparrow/buffer.hpp

```diff
@@ -438,6 +443,15 @@ namespace sparrow
         get_data().p_end = copy_initialize(first, last, get_data().p_begin, get_allocator());
     }
 
+    template <class T>
+    template <class Range, allocator A>
```

Comment:
Consider short-cutting to `template<std::ranges::input_range Range, allocator A>`

Reply:
Done